### ISSUE TITLE
Calibre OPDS recognizing cbz files as fb2

### ISSUE THREAD

**Issue by MyWebmasterD** [janeczku/calibre-web#3245]
Hello,
when downloading from Koreader in OPDS my cbz manga, the extension is changed in fb2 and Koreader is unable to recognize them
I'm using calibre web with docker

**Comment by MyWebmasterD** [comment]
yes but unfortunately I can't access that files in my docker installatioon

**Comment by BersTrayan** [comment]
> yes but unfortunately I can't access that files in my docker installatioon

Получить доступ к этим файлам в контейнере Docker можно. Для этого нужно подключиться к этому контейнеру:
`docker exec -it calibre-web bash`
Где calibre-web - название вашего контейнера для calibre-web. Узнать можно командой:
`docker ps`
Внутри выполнить команды:
`cp /app/calibre-web/cps/__init__.py /root/`
`sed 's/\/zip/\/x-cbz/' /root/__init__.py > /app/calibre-web/cps/__init__.py`
`sed 's/\/x-rar/\/x-cbr/' /root/__init__.py > /app/calibre-web/cps/__init__.py`
И перезапустить контейнер.
Правда, это придется делать после каждого обновления контейнера calibre-web, если данные изменения не внесут в основной код.

**Comment by MyWebmasterD** [comment]
It worked, thank you man!
I hope they could solve the issue in the next release

**Comment by BersTrayan** [comment]
> Unfortunately the fix brings another problem...not it is now possible to upload cbz in Calibre web..it says that is not possible to upload the file type

Обновите контейнер Calibre-web с `latest` до `nightly`, там все эти проблемы исправлены. Ни чего менять самостоятельно не придётся.

**Comment by Bilbotorm-Git** [comment]
Problem was also there for downloading CBZ file from OPDS Calibre Web in docker (Container Manager on Synology DSM7 NAS) ; the command line above fixed it for me on both KOReader and Comic Rack reader (ipad)
